spec/components/mark_complete_component/view_spec.rb

```diff
@@ -9,7 +9,7 @@ def generate_form_builder(model)
   let(:form) { build(:form, :with_pages, id: 2) }
   let(:mark_complete_form) { Forms::MarkCompleteForm.new(form:).assign_form_values }
 
-  context "when the task status feature is enabled", feature_task_list_statuses: true do
+  context "when the task status feature is enabled" do
```

Comment:
Not sure this name really makes sense any more? We can probably remove this context tag altogether